src/lib/lang/evaluators/order1/parenExpression.ts

```diff
@@ -0,0 +1,27 @@
+import type { TreeCursor } from '@lezer/common'
+import { Sequence } from '../../dataStructures'
+import { evalOrderLast } from ".."
+import { ErrorNodeError } from '../errors'
+
+    //for the paren expression we just need to go through everything within the parentheses and evaluate it FIRST before evaluating other things
+export function evalParenExpression(src: string, node: TreeCursor): number | Sequence{
+
+    // result can either evaluate to a flat number or be a sequence depending on the contents within the parentheses
+    let results: number|Sequence;
+
+    // there is only one child within the parentheses
+        //reminder: node.firstChild SETS node to the first child and it RETURNS a bool 
+        //it does not return the first child like one might initially think
+    node.firstChild() 
```

Comment:
If this is false, someone wrote a `()`. While not particularly useful to write, it's probably a behavior worth handling explicitly. Off the top of my head, I'm thinking () -> 0. That way, say 1d() -> [0] and {()} -> [0]

Reply:
Unfortunately, node.firstChild would return true even if there was no child present.